images/Dockerfile.arktosvmruntime-base

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+MAINTAINER Yunwen Bai <yunwen.bai@futurewei.com>
+
+# BUMP 23.11.2018
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN echo deb-src http://archive.ubuntu.com/ubuntu/ bionic main universe restricted >>/etc/apt/sources.list && \
+    echo deb-src http://archive.ubuntu.com/ubuntu/ bionic-updates main universe restricted >>/etc/apt/sources.list
+
+RUN apt-get -y update && apt-get -y upgrade && \
```

Comment:
It's better to combine these RUNs to reduce the layers of the image.

Reply:
yes, will do. thanks.

Reply:
fixed by re-org the docker commands and reduced some.